Gruntfile.js

```diff
@@ -43,6 +48,32 @@
  *  grunt update_json - This automates updating the bower file
  *                      to match the package.json
  */
+
+function getYuidocOptions() {
+  const BASE_YUIDOC_OPTIONS = {
+    name: '<%= pkg.name %>',
+    description: '<%= pkg.description %>',
+    version: '<%= pkg.version %>',
+    url: '<%= pkg.homepage %>',
+    options: {
+      paths: ['src/', 'lib/addons/'],
+      themedir: 'docs/yuidoc-p5-theme/',
+      helpers: [],
+      outdir: 'docs/reference/'
+    }
+  };
+
+  var o = {
+    prod: JSON.parse(JSON.stringify(BASE_YUIDOC_OPTIONS)),
+    dev: JSON.parse(JSON.stringify(BASE_YUIDOC_OPTIONS))
+  };
+
+  o.prod.options.helpers.push('docs/yuidoc-p5-theme/helpers/helpers_prod.js');
+  o.dev.options.helpers.push('docs/yuidoc-p5-theme/helpers/helpers_dev.js');
+
+  return o;
+}
```

Comment:
This isn't super ideal, but it's the only way I could think of improving the DRYness of the yuidoc stuff. :unamused: Hopefully it's decently readable?